ui/imports/shared/views/AssetsDetailView.qml

```diff
@@ -18,8 +19,78 @@ Item {
 
     property var token
 
-    signal goBack()
+    QtObject {
+        id: d
+        //dummy data
+        property real stepSize: 1000
+        property real minStep: 12000
+        property real maxStep: 22000
+
+        property var graphTabsModel: [{text: "Price", enabled: true}, {text: "Balance", enabled: false}]
+        property var timeRangeTabsModel: [{text: "1H", enabled: true},
+                {text: "1D", enabled: true},{text: "7D", enabled: true},
+                {text: "1M", enabled: true}, {text: "6M", enabled: true},
+                {text: "1Y", enabled: true}, {text: "ALL", enabled: true}]
```

Comment:
Here you need to add the `qsTr()` for `text`s :)

Reply:
added